frontend-react/src/hooks/UseOrganizationReceiversFeed.ts

```diff
@@ -18,7 +18,12 @@ export const useOrganizationReceiversFeed = (): ReceiverFeeds => {
 
     useEffect(() => {
         if (receivers?.length) {
-            setActive(receivers[0]);
+            setActive(
+                // Checks for an active receiver first
+                receivers.find((val) => val.customerStatus === "active") ||
```

Comment:
Can we use [the enum](https://github.com/CDCgov/prime-reportstream/blob/master/frontend-react/src/utils/TemporarySettingsAPITypes.ts#L18) instead of a hardcoded string?

Reply:
Oh snap, didn't know we had that merged! Will do